fiware-region-sanity-tests/sanity_checks

```diff
@@ -86,7 +86,7 @@ export OS_TENANT_ID OS_TENANT_NAME OS_USER_DOMAIN_NAME
 
 # Options for nosetests
 TESTS=
-NOSEOPTS="--logging-filter=TestCase,novaclient,neutronclient --logging-level=ERROR"
+NOSEOPTS="--config=.noserc"
```

Comment:
I'd prefer naming the configuration file as `nose.cfg`, given that files `.*rc` are usually reserved from $HOME directories.